### PR TITLE
some corrections in the CL-> Utilities

### PR DESCRIPTION
**Pull request by AlejandroCornejo**
Hello! pls Tobias/Vicente tell me if I've made more mistakes haha
Thank you

### CODE REVIEW

File: applications/StructuralMechanicsApplication/custom_utilities/constitutive_law_utilities.cpp [hunk 1]
```diff
@@ -567,15 +567,13 @@ void ConstitutiveLawUtilities<6>::CalculateProjectionOperator(
...
-    indexes_ij(2, 0) = 0;
-    indexes_ij(2, 1) = 2;
 
     IndexType i, j;
     double h_i = 0.0, h_j = 0.0;
     Matrix cross_p_ij_tensor;
     Vector cross_p_ij_vector;
 
-    for (IndexType index = 0; index < Dimension; ++index) {
+    for (IndexType index = 0; index < 2; ++index) {
```
Comment (tteschemacher) [inline applications/StructuralMechanicsApplication/custom_utilities/constitutive_law_utilities.cpp]: of maybe directly?
for (IndexType i=0; i< Dimension; ++i) {
for (IndexType j=i; j< Dimension; ++j) {

File: applications/StructuralMechanicsApplication/custom_utilities/constitutive_law_utilities.cpp [hunk 2]
```diff
@@ -567,15 +567,13 @@ void ConstitutiveLawUtilities<6>::CalculateProjectionOperator(
     indexes_ij(0, 1) = 1;
     indexes_ij(1, 0) = 1;
     indexes_ij(1, 1) = 2;
-    indexes_ij(2, 0) = 0;
-    indexes_ij(2, 1) = 2;
```
Comment (tteschemacher) [inline applications/StructuralMechanicsApplication/custom_utilities/constitutive_law_utilities.cpp]: Sorry I think this was right before.

File: applications/StructuralMechanicsApplication/custom_utilities/constitutive_law_utilities.cpp (hunk 1 above)
Comment (tteschemacher) [reply applications/StructuralMechanicsApplication/custom_utilities/constitutive_law_utilities.cpp]: wait this also does not work. It should be something like this. This Sum is very confusing...
for (IndexType i=0; i< Dimension - 1; ++i) {
for (IndexType j=i + 1; j< Dimension; ++j) {

File: applications/StructuralMechanicsApplication/custom_utilities/constitutive_law_utilities.cpp (hunk 2 above)
Comment (AlejandroCornejo) [reply applications/StructuralMechanicsApplication/custom_utilities/constitutive_law_utilities.cpp]: yes! I think that this: 

for (IndexType i=0; i< Dimension - 1; ++i) {
for (IndexType j=i + 1; j< Dimension; ++j) {

is the same that the code in the begining.

**Review by loumalouomega — DISMISSED** [review]

**Comment by AlejandroCornejo** [comment]
with this push should be everything right

**Comment by tteschemacher** [comment]
Looks good to me. Thanks!

**Review by tteschemacher — APPROVED** [review]

**Review by loumalouomega — APPROVED** [review]

**Review by frastellini — APPROVED** [review]
Fine.

**Comment by AlejandroCornejo** [comment]
thank you @frastellini :)

**Comment by AlejandroCornejo** [comment]
Why it takes so long to pass the travis???

**Comment by loumalouomega** [comment]
@roigcarlo

**Comment by roigcarlo** [comment]
It has passed. but seems like github did not get the hook. I restarted the build.

**Pull request merged**